src/test/ui/unboxed-closures/unboxed-closure-sugar-used-on-struct.stderr

```diff
@@ -2,15 +2,18 @@ error[E0214]: parenthesized parameters may only be used with a trait
   --> $DIR/unboxed-closure-sugar-used-on-struct.rs:7:18
    |
 LL | fn foo(b: Box<Bar()>) {
-   |                  ^^ only traits may use parentheses
+   |                  ^^
+   |                  |
+   |                  only traits may use parentheses
+   |                  help: use angle brackets instead: `<>`
```

Comment:
Hm, this is suboptimal, though; we'd prefer to cut the parameter list entirely rather than suggest `<>`.